README.md

```diff
@@ -1,8 +1,8 @@
 # How to Get Started with the Syncfusion Vue Grid
 
-A quick start project that helps you to create and bind data to the Syncfusion Vue Grid. You will also learn how to customize tablular data and enable few of its important features like paging, sorting, filtering and grouping.
+The [Vue Grid](https://www.syncfusion.com/vue-components/vue-grid?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples), is a feature-rich control for displaying data in a tabular format. Its wide range of functionalities includes data binding, editing, Excel-like filtering, custom sorting, grouping, row reordering, freezing rows and columns, exporting to Excel, CSV and PDF formats, aggregating rows and more. A quick start project that helps you to create and bind data to the Syncfusion Vue Grid. You will also learn how to customize tablular data and enable few of its important features like paging, sorting, filtering and grouping.

```

Comment:
Recheck on this. Remove comma after UTM parameter link and also no need for additional explanation.
![image](https://github.com/SyncfusionExamples/getting-started-with-vue-data-grid-of-syncfusion/assets/95272441/437d06cb-56f8-49fb-ac09-9cc0606894fe)

Reply:
Done